docs/cubeviz/index.rst

```diff
@@ -0,0 +1,70 @@
+
+.. image:: ../logos/cubeviz.svg
+   :width: 400
+
+#######
+CubeViz
+#######
+
+CubeViz is an visualization and analysis toolbox for data cubes from integral field units (IFUs). It is built as part of the glue visualiztion tool. CubeViz is designed to work with data cubes from the NIRSpec and MIRI instruments on JWST, and will work with data cubes from any IFU. It uses the `specutils <https://specutils.readthedocs.io/en/latest/>`_ package from `Astropy <https://www.astropy.org>`_ .
```

Comment:
"a visualization"
"glue visualization" (sp.)
"will work with data cubes from any IFU"   Will it?